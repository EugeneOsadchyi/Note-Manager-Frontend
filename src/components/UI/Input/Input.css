.Input {
  display: block;
  border: 2px solid black;
  line-height: 25px;
  margin: 0 auto;
  width: 60%;
  margin-bottom: 20px;
  padding: 2px 7px;
}
