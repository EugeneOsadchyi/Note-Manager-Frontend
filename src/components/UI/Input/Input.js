import React from 'react';
import PropTypes from 'prop-types';
import './Input.css';

const Input = ({ placeholder }) => (
  <input className="Input" type="text" placeholder={placeholder} />
);

Input.propTypes = {
  placeholder: PropTypes.string,
};

Input.defaultProps = {
  placeholder: '',
};

export default Input;
