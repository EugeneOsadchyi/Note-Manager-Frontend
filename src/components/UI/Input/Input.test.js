import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Input from './Input';

describe('Input', () => {
  const input = shallow(<Input />);

  it('renders properly', () => {
    expect(shallowToJson(input)).toMatchSnapshot();
  });
});
