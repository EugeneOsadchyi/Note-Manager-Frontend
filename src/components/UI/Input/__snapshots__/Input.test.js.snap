// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input renders properly 1`] = `
<input
  className="Input"
  placeholder=""
  type="text"
/>
`;
